docs/content/ai_integrations/cohere.md

```diff
@@ -1,6 +1,6 @@
 # Cohere
 
-`superduper` allows users to work with `cohere` API models.
+`Superduper` allows users to work with `cohere` API models.
```

Comment:
This is the package.

Reply:
Noted. The package will be package. I will keep the superduper as it is where it mentions the package.